src/types/config.ts

```diff
@@ -1,8 +1,19 @@
-import { existsSync } from 'fs'
-import { dirname, resolve } from 'path'
-import { fileURLToPath } from 'url'
+import { isNode } from '../util'
 import { Logger, consoleLogger, nullLogger } from './logging'
 
+let fs: typeof import('fs')
```

Comment:
Rather than using global variables you could inline them

i.e. const fs = await import(“fs”)